cmd/podman/machine/cp.go

```diff
@@ -0,0 +1,137 @@
+//go:build amd64 || arm64
+
+package machine
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	"github.com/containers/podman/v5/libpod/events"
+	"github.com/containers/podman/v5/pkg/copy"
+	"github.com/containers/podman/v5/pkg/machine"
+	"github.com/containers/podman/v5/pkg/machine/define"
+	"github.com/containers/podman/v5/pkg/machine/env"
+	"github.com/containers/podman/v5/pkg/machine/vmconfigs"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cpCmd = &cobra.Command{
+		Use:               "cp [options] [MACHINE:]SRC_PATH [MACHINE:]DEST_PATH",
+		Short:             "Securely copy contents between the virtual machine",
+		Long:              "Securely copy files or directories between the virtual machine and your host",
+		PersistentPreRunE: machinePreRunE,
+		RunE:              cp,
+		Args:              cobra.ExactArgs(2),
+		Example:           `podman machine cp ca.crt podman-machine-default:/etc/containers/certs.d/ca.crt`,
+		ValidArgsFunction: completion.AutocompleteNone,
```

Comment:
please add a valid shell completion function, AutocompleteNone is effectively user hostile as you do not allow them to get shell completion even for local paths.